mutiny-core/src/nodemanager.rs

```diff
@@ -739,15 +797,65 @@ impl NodeManager {
     /// Lists all the on-chain transactions in the wallet.
     /// These are sorted by confirmation time.
     pub fn list_onchain(&self) -> Result<Vec<TransactionDetails>, MutinyError> {
-        let mut txs = self.wallet.list_transactions(false)?;
+        let mut txs = self.wallet.list_transactions(true)?;
         txs.sort();
+        let address_labels = self.get_address_labels()?;
+        let txs = txs
+            .into_iter()
+            .map(|tx| {
+                let labels = tx
+                    .transaction
+                    .clone()
+                    .unwrap()
```

Comment:
A comment on this `unwrap` would be nice, knowing that it's only safe when `list_transactions` is true.